### DISCUSSION TITLE
How to use hydro with Bootstrap modal, accordion, and any bootstrap JavaScript related

### DISCUSSION THREAD

**Discussion by PatrickPo87** [hydrostack/hydro#34] [Q&A] [answered]
Hi Guys! Im having fun with hydro. I would like to ask for your how to use bootstrap modals, accordion, tostrjs or sweetalert or jquery ui datepicker with hydro? Thanks in advance

**Comment by kjeske** [comment, marked as answer]
Hi! Thank for trying Hydro!

When it comes to usage of JS UI libraries, you can use build-in AlpineJs directives to trigger external JS code. Especially [init](https://alpinejs.dev/directives/init) is useful here. Once the element is loaded on page you can run custom JS code that will trigger your external libraries.

```html
<div x-init="$($el).dialog()">
  Content
</div>
```

**Reply by PatrickPo87** [reply]
Thank you @kjeske ! big help. I really like Hydro =)

**Reply by kjeske** [reply]
Glad to hear! 😄
